code/__DEFINES/dcs/signals/signals_atom/signals_atom.dm

```diff
@@ -9,6 +9,8 @@
 #define COMSIG_PARENT_EXAMINE "atom_examine"
 /// from base of atom/get_examine_name(): (/mob, list/overrides)
 #define COMSIG_ATOM_GET_EXAMINE_NAME "atom_examine_name"
+///from base of atom/examine(): (/mob, list/examine_text)
+#define COMSIG_ATOM_EXAMINE "atom_examine"
 	//Positions for overrides list
 	#define EXAMINE_POSITION_ARTICLE (1<<0)
 	#define EXAMINE_POSITION_BEFORE (1<<1)
```

Comment:
The indented defines are actually the return values of signal calls. They should always be below the signal that actually listens to their returns. In this case `EXAMINE_POSITION_ARTICLE` does not belong to `COMSIG_ATOM_EXAMINE`, so it shouldn't be underneath it and it should be under `COMSIG_ATOM_GET_EXAMINE_NAME` since thats the signal that uses it.

Reply:
Fixed